.gitignore

```diff
@@ -11,6 +11,7 @@ logs/
 .idea/
 
 # Binaries for programs and plugins
+/src/sdmm
```

Comment:
```suggestion
./src/sdmm
```

Reply:
With a leading `.`, the pattern doesn't actually match the file. A leading `/` [is correct](https://git-scm.com/docs/gitignore#_pattern_format):

> If there is a separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular .gitignore file itself. Otherwise the pattern may also match at any level below the .gitignore level.